### PR TITLE
Update TOPP.doxygen

### PR DESCRIPTION
**Pull request by timosachsenberg**
## Checklist
- [ ] Make sure that you are listed in the AUTHORS file
- [ ] Add relevant changes and new features to the CHANGELOG file
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] New and existing unit tests pass locally with my changes
- [ ] Updated or added python bindings for changed or new classes (Tick if no updates were necessary.)

### How can I get additional information on failed tests during CI
<details>
  <summary>Click to expand</summary>
If your PR is failing you can check out

- The details of the action statuses at the end of the PR or the "Checks" tab.
- http://cdash.openms.de/index.php?project=OpenMS and look for your PR. Use the "Show filters" capability on the top right to search for your PR number.
  If you click in the column that lists the failed tests you will get detailed error messages.

</details>

### Advanced commands (admins / reviewer only)
<details>
  <summary>Click to expand</summary>
  
- `/reformat` (experimental) applies the clang-format style changes as additional commit. Note: your branch must have a different name (e.g., yourrepo:feature/XYZ) than the receiving branch (e.g., OpenMS:develop). Otherwise, reformat fails to push.
- setting the label "NoJenkins" will skip tests for this PR on jenkins (saves resources e.g., on edits that do not affect tests)
- commenting with `rebuild jenkins` will retrigger Jenkins-based CI builds
  
</details>

---
:warning: Note: Once you opened a PR try to minimize the number of *pushes* to it as every push will trigger CI (automated builds and test) and is rather heavy on our infrastructure (e.g., if several pushes per day are performed).

### CODE REVIEW

**Comment by timosachsenberg** [comment]
I gave it a first round... please add/change/modify or comment

File: doc/doxygen/public/TOPP.doxygen [hunk 1]
```diff
@@ -46,33 +46,31 @@
...
-  - @subpage TOPP_NoiseFilterGaussian - Removes noise from profile spectra by using different smoothing techniques.
-  - @subpage TOPP_NoiseFilterSGolay - Removes noise from profile spectra by using different smoothing techniques.
+  <b>Centroiding</b>
   - @subpage TOPP_PeakPickerHiRes - Finds mass spectrometric peaks in profile mass spectra.
   - @subpage TOPP_PeakPickerWavelet - Finds mass spectrometric peaks in profile mass spectra.
-  - @subpage TOPP_PrecursorMassCorrector - Correct the precursor entries of tandem MS scans.
-  - @subpage TOPP_HighResPrecursorMassCorrector - Correct the precursor entries of tandem MS scans for high resolution data.
-  - @subpage TOPP_Resampler - Transforms an LC-MS map into an equally-spaced (in RT and m/z) map.
+
+  <b>Sprectra filtering</b>
```
Comment (jpfeuffer) [inline doc/doxygen/public/TOPP.doxygen:73]:
```suggestion
  <b>Spectrum processing: peak filtering</b>
```

File: doc/doxygen/public/TOPP.doxygen [hunk 2]
```diff
@@ -82,15 +80,46 @@
   - @subpage TOPP_SpectraFilterSqrtMower - Applies a filter to peak spectra.
   - @subpage TOPP_SpectraFilterThresholdMower - Applies a filter to peak spectra.
   - @subpage TOPP_SpectraFilterWindowMower - Applies a filter to peak spectra.
-  - @subpage TOPP_SpectraMerger - Merges spectra from an LC-MS map, either by precursor or by RT blocks.
+  - @subpage TOPP_BaselineFilter - Removes the baseline from profile spectra using a top-hat filter.
+  - @subpage TOPP_NoiseFilterGaussian - Removes noise from profile spectra by using different smoothing techniques.
+  - @subpage TOPP_NoiseFilterSGolay - Removes noise from profile spectra by using different smoothing techniques.
+  - @subpage TOPP_MapNormalizer - Normalizes peak intensities in an MS run.
```
Comment (jpfeuffer) [inline doc/doxygen/public/TOPP.doxygen:86]:
```suggestion
<b>Spectrum processing: peak normalization</b>
  - @subpage TOPP_MapNormalizer - Normalizes peak intensities in an MS run.  
```

File: doc/doxygen/public/TOPP.doxygen [hunk 3]
```diff
@@ -82,15 +80,46 @@
...
+  - @subpage TOPP_MapNormalizer - Normalizes peak intensities in an MS run.  
+  
+  <b>Mass Correction and Calibration</b>
+  - @subpage TOPP_InternalCalibration - Applies an internal mass calibration.
+  - @subpage TOPP_ExternalCalibration - Applies an external mass calibration.
+  - @subpage TOPP_HighResPrecursorMassCorrector - Correct the precursor entries of tandem MS scans for high resolution data.
+  - @subpage TOPP_PrecursorMassCorrector - Correct the precursor entries of tandem MS scans (low-res only).
   - @subpage TOPP_TOFCalibration - Applies time of flight mass calibration.
+  
+  <b>Misc Spectrum Processing Tools</b>
```
Comment (jpfeuffer) [inline doc/doxygen/public/TOPP.doxygen:95]:
```suggestion
  <b>Spectrum processing: Misc</b>
```

**Pull request merged**